.packit.yaml

```diff
@@ -19,6 +19,8 @@ srpm_build_deps:
 #   post-upstream-clone: "./tools/rpm_spec_add_provides_bundle.sh"
 
 actions:
+  get-current-version:
```

Comment:
this was due some changes which was pulling the version from the release tag(0.15.4) instead of the version in spec, so had to extract it from spec file.